misc/make_tests.py

```diff
@@ -0,0 +1,113 @@
+# Generate unit test file for Graphic3d_RenderingParams
```

Comment:
Do we need this file in the repo? Nice script to have locally, but once the main test file is created I think I would just rather edit it manually. Unless I'm missing something maybe we leave this file out?

Reply:
My thinking is that it would be useful for other developers who need to do something similar in the future. Perhaps it would be worth having a directory called something like "utils" or "tools", for things that aren't needed for the actual build but which are useful for development, maintenance etc.

Anyway, I've deleted it and stashed it safely away on my local machine :smiley: